### PR TITLE
PIM-6806: Add new job instance in the changelog

### PR DESCRIPTION
**Pull request by SamirBoulil**
**Description (for Contributor and Core Developer)**

Introducing features in patches sometimes induces adding new jobs. We use to do doctrine migration for new jobs but as discussed with @nidup for minor versions we prefer to document this change in the changelog.

A new internal job as been introduced here https://github.com/akeneo/pim-community-dev/pull/7006. 

This PR documents it in the changelog. 

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added Behats                      | -
| Added integration tests           | -
| Changelog updated                 | Ok
| Review and 2 GTM                  | Ok
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: CHANGELOG-2.0.md [hunk 1]
```diff
@@ -17,6 +17,9 @@
 - Change the constructor of `Pim\Component\Catalog\Completeness\CompletenessCalculator`. Remove `Pim\Component\Catalog\Factory\ValueFactory` and both `Akeneo\Component\StorageUtils\Repository\CachedObjectRepositoryInterface`. Add `Pim\Component\Catalog\EntityWithFamily\IncompleteValueCollectionFactory` and `Pim\Component\Catalog\EntityWithFamily\RequiredValueCollectionFactory`.
 - Change the constructor of `Pim\Bundle\EnrichBundle\Normalizer\ProductModelNormalizer` to add `Symfony\Component\Serializer\Normalizer\NormalizerInterface`.
 
+## New jobs
```
Comment (SamirBoulil) [inline CHANGELOG-2.0.md]: I know the "null" is not the best, but our constraint NOT_BLANK won't let me create a jobinstance with an empty job parameter :/ Any idea @nidup ?

Comment (nidup) [reply CHANGELOG-2.0.md]: hum, sounds the less worse approach, keep in mind to add something in EE for the exec permissions of this new job instance

Comment (SamirBoulil) [reply CHANGELOG-2.0.md]: right

File: CHANGELOG-2.0.md [hunk 2]
```diff
@@ -17,6 +17,9 @@
...
+- Add the job instance `compute_completeness_of_products_family` (`bin/console akeneo:batch:create-job "internal" "compute_completeness_of_products_family" "compute_completeness_of_products_family" "compute_completeness_of_products_family" '{"family_code":"null"}' "compute completeness of products family" --env=prod`
```
Comment (grena) [inline CHANGELOG-2.0.md]: missing 2nd parenthesis?

Comment (SamirBoulil) [reply CHANGELOG-2.0.md]: 🐯

**Pull request merged**